packages/components/src/PrimaryToolbar/PrimaryToolbar.js

```diff
@@ -149,6 +151,7 @@ PrimaryToolbar.propTypes = {
     PropTypes.shape({
       onClick: PropTypes.func,
       isAllExpanded: PropTypes.bool,
+      isDisabled: PropTypes.bool
```

Comment:
Can we add `buttonProps` as well?
```suggestion
      isDisabled: PropTypes.bool,
      buttonProps: PropTypes.shape({
        [PropTypes.string]: PropTypes.object
       })
```